.github/workflows/ci.yaml

```
@@ -25,8 +25,10 @@ jobs:
       matrix:
         ruby:
           - "2.7"
+          - "3.0"
         puppet:
           - "7"
+          - "8"
```

You can exclude Ruby 2.7 & Puppet 8, since it's an invalid combination:
```suggestion
          - "8"
        exclude:
          - ruby: "2.7"
            puppet: "8"
```

---

You can also explicitly write out the matrix:
```yaml
include:
  - ruby: '2.7'
    puppet: '7'
  - ruby: '3.2'
    puppet: '8'